samples/unit-tests/series/datasorting/demo.js

```diff
@@ -95,3 +95,54 @@ QUnit.test('Data sorting ', function (assert) {
     );
 
 });
+
+QUnit.test('Data sorting with sortKey', function (assert) {
+
+    Highcharts.chart('container', {
+        series: [{
+            type: 'column',
+            data: [{
+                custom: {
+                    myValue: 'b'
+                },
+                y: 3
+            }, {
+                custom: {
+                    myValue: 'c'
+                },
+                y: 1
+            }, {
+                custom: {
+                    myValue: 'a'
+                },
+                y: 2
+            }],
+            dataSorting: {
+                enabled: true
+            }
+        }]
+    }, function (chart) {
```

Comment:
Just a question: Any particular reason to do the test in callback?

Reply:
I do not think so. I guess, I used it just because of the similarity to the TestTemplate pattern.